pyopnsense/routes.py

```diff
@@ -0,0 +1,18 @@
+from pyopnsense import client
```

Comment:
Can you add a license statement to this file (you can use your copyright line if you want) but I think we should include the same gpl header as the other modules. For example:

https://github.com/mtreinish/pyopnsense/blob/master/pyopnsense/firmware.py#L3-L16

Reply:
Done